dp/cloud/python/magma/fluentd_client/config.py

```diff
@@ -0,0 +1,65 @@
+"""
+Copyright 2021 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import importlib
+import os
+from distutils.util import strtobool
+
+
+class Config(object):
+    """
+    Configuration class for Configuration Controller
+    """
+    # General
+    LOG_LEVEL = os.environ.get('LOG_LEVEL', 'DEBUG')
+
+    # Elasticsearch
+    ELASTICSEARCH_INDEX = os.environ.get('ELASTICSEARCH_INDEX', 'dp')
+
+    # Fluentd
```

Comment:
It looks like the orchestrator configuration variables that define where orc8r fluentd is are: `FLUENTD_SERVICE_HOST` and (`FLUENTD_SERVICE_PORT` or `FLUENTD_SERVICE_PORT_FORWARD`)